Rubberduck.CodeAnalysis/Inspections/Concrete/EmptyBlockInspectionListenerBase.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Rubberduck.Inspections.Concrete
 {
-    public class EmptyBlockInspectionListenerBase<TContext> : InspectionListenerBase<TContext>
+    internal class EmptyBlockInspectionListenerBase<TContext> : InspectionListenerBase<TContext>
```

Comment:
Would it be cleaner to have an abstract `EmptyBlockInspectionBase<TContext>` with a `protected EmptyBlockInspectionListenerBase<TContext> ContextListener { get; }`? That way the listener becomes private to the `EmptyBlockInspectionBase`.

Reply:
The `Base` suffix says it already, but having this an `abstract` class would make it compiler-enforced that it's not intended to be instantiated directly, too.